react/react1/week1/react1week1/src/TodoList.jsx

```diff
@@ -0,0 +1,39 @@
+const todoList = [
+    {
+      id: 0,
+      description: "Get out of bed",
+      deadLineDate: "Wed Sep 13 2017",
+    },
+    {
+      id: 1,
+      description:"Brush teeth" ,
+      deadLineDate: "Thu Sep 14 2017",
+     
+    },
+    {
+        id: 2,
+        description:"Eat breakfast" ,
+        deadLineDate: "Fri Sep 15 2017",
+       
+      },
+  ];
+export function TodoList() {
+    return(
+    <>
+   
+        {todoList.map((item) => <TodoListItems descp={item.description} date={item.deadLineDate} key={item.id}/> )}
```

Comment:
those props aren't relevant here, use them inside the component

```suggestion
        {todoList.map((item) => <TodoListItems item={item} key={item.id}/> )}
```